deploy/manifests/base/indexstar/pod-monitor.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Include `pod-monitor.yaml` in `kustomization.yaml` at this directory under `resources`.